modules/storages/app/contracts/storages/storages/one_drive_contract.rb

```diff
@@ -35,6 +35,8 @@ class OneDriveContract < ::ModelContract
     attribute :tenant_id
     validates :tenant_id, format: { with: /\A(?:[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}|consumers)\z/i }
     attribute :drive_id
-    validates :drive_id, presence: true, allow_nil: true
+    validates :drive_id,
+              presence: { message: I18n.t('activerecord.errors.models.storages/storage.attributes.drive_id.id_blank') },
```

Comment:
❓ why do we need a specific localized message for `drive_id` and not for `tenant_id`? according to your screen, the message for tenant contains the id, but for drive it does not (before your fix). How is that?

Reply:
active model errors strips out the "ID" from the message when the attribute name has an "_id" suffix.

Reply:
> ❓ why do we need a specific localized message for `drive_id` and not for `tenant_id`? according to your screen, the message for tenant contains the id, but for drive it does not (before your fix). How is that?

Ah, indeed odd- let me compare! Could do without additional translations 👀